src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/CreateNamespaceCommand.java

```diff
@@ -12,7 +12,12 @@
 import javax.annotation.Nullable;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-@Schema(description = "Command that creates a namespace.")
+/**
+ * Command {@link CreateNamespaceCommand} is deprecated, please switch to {@link
+ * CreateKeyspaceCommand} Support it for backward-compatibility
+ */
+@Schema(
+    description = "Command that creates a namespace.(Deprecated, please switch to createKeyspace.)")
```

Comment:
nit/formatting: should be formatted like:

description = "Command that creates a namespace (deprecated, please switch to createKeyspace).")

Reply:
solved, thanks